javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -194,8 +215,25 @@ object Character {
   final val MIN_SUPPLEMENTARY_CODE_POINT = 0x10000
 
   // Not implemented:
-  //def getType(ch: scala.Char): scala.Int
-  //def getType(codePoint: scala.Int): scala.Int
+  def getType(ch: scala.Char): scala.Int = getType(ch.toInt)
+
+  def getType(codePoint: scala.Int): scala.Int = {
+    if (codePoint < 256) getTypeTL256(codePoint)
+    else getTypeGTE256(codePoint)
```

Comment:
`LT` and `GE` are more customary acronyms for lower-than and greater-or-equal in this codebase.